Resources/views/SubscriptionWall/edit.html.twig

```diff
@@ -0,0 +1,24 @@
+{% extends 'IntegratedContentBundle::base.html.twig' %}
+
+{% block content %}
+    <div id="wrapper-holder">
+        <div class="container">
+            <div class="content_columns_holder">
+                <div class="content_column_inner">
+                    <div class="results">
+                        {% for flash_message in app.session.flashBag.get('notice') %}
```

Comment:
shy do you show "notice", I don't see a message anywhere in this commit with "notice"